sdk/resourcemanager/Azure.ResourceManager.Core/tests/Unit/TestAssets/ArmException/ArmExceptionSimpleV1.json

```diff
@@ -0,0 +1,24 @@
+{
+    "code": "400",
```

Comment:
should we have a few more cases like

- required things missing and assert the proper serialization failure
- missing opitional elements and assert nulls / empties
- etc